frontstage/controllers/party_controller.py

```diff
@@ -510,3 +511,39 @@ def notify_party_and_respondent_account_locked(respondent_id, email_address, sta
         raise ApiError(logger, response)
 
     bound_logger.info('Successfully notified respondent and party service that account is locked')
+
+
+def get_list_of_business_for_party(party_id):
```

Comment:
Docstring here would do a lot of work.  
What party_id is it?  We still live in a world where a party_id can be a business or a respondent.

Reply:
done